app/static/src/app/store/translations/registerTranslations.ts

```diff
@@ -0,0 +1,16 @@
+import { Store } from "vuex";
+import i18next from "i18next";
+import { Language, locales } from "./locales";
+import { AppState } from "../appState/state";
+
+export default <S extends AppState>(store: Store<S>) => {
+    i18next.init({
```

Comment:
I thought you were intended to spread the translation strings across different folders? So rather than doing a monolithic init here, could potentially `useResourceBundle` from each relevant folder (though would have to watch when that actually got called), or alternative coordinate registering all those locations from here.

Also - is there anything to be done here around whether `internationalisation` is false. If that's the case, do we only need to register translations for the default language (and the fallback language?)